src/main/java/racingcar/view/OutputView.java

```diff
@@ -0,0 +1,21 @@
+package racingcar.view;
+
+public class OutputView implements View {
+
+    private final String winners;
+
+    public OutputView(String winners) {
+        this.winners = winners;
+    }
+
+    private void printWinners() {
+        String announcement = "최종 우승자 : ";
+        System.out.println(announcement + winners);
+    }
+
+    @Override
+    public String printView() {
+        printWinners();
+        return null;
+    }
```

Comment:
```suggestion
    private void printWinners(String winners) {
        String announcement = "최종 우승자 : ";
        System.out.println(announcement + winners);
    }

    @Override
    public String printView(String winners) {
        printWinners(winners);
        return null;
    }
```
궁금한 점이 있습니다.. 
이렇게 OutputView가 필드로 갖지 않고, 메서드에 파라미터로 winners를 주입받는 방식과 어떤 점이 다른 걸까요??
어떤 게 옳은 건지, 아니면 취향 차이인 건지 궁금합니다.

Reply:
제가 View 클래스들을 `View` 인터페이스를 모두 구현하도록 해서 `printView()` 메소드로 최종 출력을 할 수 있도록 구상했습니다!

그래서 오버라이딩한 `printView()` 메소드가 파라미터를 받지 않아서 `OutputView`가 필드로 `winners`를 가지고 출력하도록 하였습니다!